src/node_nanomsg.cc

```diff
@@ -37,7 +39,11 @@ NAN_METHOD(Close) {
     rc = nn_close(s);
   } while (rc < 0 && errno == EINTR);
 
-  info.GetReturnValue().Set(Nan::New<Number>(rc));
+  v8::Local<v8::Number> ret = Nan::New<Number>(rc);
+  Local<Value> argv[] = { ret };
```

Comment:
That's the standard way to fill values given to v8 callbacks, I think we have to create this array even if we're going to only return one value as the parameter to the callback